roles/openshift_logging/templates/kibana.j2

```diff
@@ -44,15 +44,19 @@ spec:
 {% if kibana_cpu_limit is not none %}
               cpu: "{{kibana_cpu_limit}}"
 {% endif %}
-{% if kibana_memory_limit is not none %}
```

Comment:
@ewolinetz I'd suggested it under the assumption that since it had changed from null to an actual default it was no longer necessary because it would always have a value. I'm fine with the more recent changes too.